src/components/Calculator.js

```diff
@@ -1,21 +1,61 @@
-export default function Calculator() {
+import { useState } from 'react';
+import calculate from '../logic/calculate';
+
+const obj = {
+  total: null,
+  next: null,
+  operation: null,
+};
+
+function Calculator() {
+  const [output, setOutput] = useState(obj);
+
   const buttonKeys = ['AC', '+/-', '%', '÷', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', '='];
 
+  const clickBtn = (e) => {
+    const btn = e.target.name;
+    return setOutput(calculate(output, btn));
+  };
+
+  const display = (item) => {
+    if ((!item.total) && (!item.next)) {
+      return '0';
+    }
+    if (!item.total) {
+      return item.next;
+    }
+    if (item.next && item.total && item.operation) {
+      return item.next;
+    }
+    return item.total;
+  };
+
   return (
     <section id="calculator">
       <input
         id="terminate"
         type="text"
-        value="0"
+        value={display(output)}
```

Comment:
- [ ] Dear @joyapisi, as mentioned by the previous reviewer, we need to practice the usage of `Props` in this milestone. As part of that kindly ensure that Props are passed correctly to any child components of the Calculate.js component.
> Consider creating a child component and passing props to it.